platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheUnsupportedTypesIntegrationTest.groovy

```diff
@@ -17,6 +17,7 @@
 package org.gradle.configurationcache
 
 import org.gradle.api.DefaultTask
+import org.gradle.api.IsolatedProject
```

Comment:
🤔 Looking at the import I realized we placed the new type in a very crowded package.

Wouldn't it be better to choose a more confined space for it? Some ideas:
- `org.gradle.api.project`
- `org.gradle.api.isolated`

Reply:
Good point.